gtk/src/default/gtk-4.0/_tweaks.scss

```diff
@@ -23,16 +17,7 @@ windowcontrols {
         
         &.close {
             > image {
-                color: $accent_fg_color;
-                background: $accent_bg_color;
-
-                &:hover {
-                    background: gtkmix($accent_bg_color, "#fff", 90%);
-                }
-
-                &:active {
-                    background: gtkmix($accent_bg_color, "#000", 90%);
-                }
+                background: $button_color;
 
                 &:backdrop {
                     filter: saturate(0);
```

Comment:
This line can be removed, it was needed for the orange close button to looks grey in backdrop.